src/utils/exec.c

```diff
@@ -304,15 +325,33 @@ gboolean bd_utils_exec_and_capture_output (const gchar **argv, const BDExtraArg
     task_id = log_running (argv);
     success = g_spawn_sync (NULL, (gchar **) argv, NULL, G_SPAWN_SEARCH_PATH,
                             (GSpawnChildSetupFunc) set_c_locale, NULL,
-                            &stdout_data, &stderr_data, &status, error);
-    log_out (task_id, stdout_data, stderr_data);
-    log_done (task_id, status);
+                            &stdout_data, &stderr_data, &exit_status, error);
 
     if (!success) {
         /* error is already populated */
         return FALSE;
```

Comment:
Should we also add `g_free(stdout_data)` and the same for stderr here?

Reply:
Good catch, fixed.